omod-1.11/src/test/java/org/openmrs/module/webservices/rest/web/RestTestConstants1_11.java

```diff
@@ -12,4 +12,6 @@
 public class RestTestConstants1_11 {
 	
 	public static final String DRUG_UUID = "05ec820a-d297-44e3-be6e-698531d9dd3f";
+	
```

Comment:
Is this used anywhere?

Reply:
Yes, it's been used in the test files

Reply:
Can you point me to the exact line?

Reply:
https://github.com/larrystone/openmrs-module-webservices.rest/blob/41b70a6fe09f2601ed7be96c2204ea8a73bea097/omod-1.11/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_11/EncounterRoleResource1_11Test.java#L52

https://github.com/larrystone/openmrs-module-webservices.rest/blob/41b70a6fe09f2601ed7be96c2204ea8a73bea097/omod-1.11/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_11/EncounterRoleController1_11Test.java#L33